tests/approaches/test_grammar_search_invention_approach.py

```diff
@@ -577,6 +577,10 @@ def test_exact_energy_score_function():
     assert all_included_s < none_included_s  # good!
     assert all_included_s < gripperopen_excluded_s  # good!
     # Test that the score is inf when the operators make the data impossible.
+    # Note: this test will crash pyperplan's implementation of LM-Cut, so
```

Comment:
This is harmless. It's just that lmcut crashes when there's a predicate named in the goal that doesn't appear in any of the task's facts (i.e., the universe of delete-relax-reachable facts). That's true for this test, because it's a hacky test, but it won't be true for predicate invention. And if we ever need it to be, we can add a light wrapper to our pyperplan glue.